node/core/pvf/src/prepare/worker.rs

```diff
@@ -169,6 +166,77 @@ pub async fn start_work(
 	.await
 }
 
+/// Handles receiving response bytes on the host from the child.
+async fn handle_response_bytes(
+	response_bytes: Vec<u8>,
+	pid: u32,
+	tmp_file: PathBuf,
+	artifact_path: PathBuf,
+	preparation_timeout: Duration,
+) -> Selected {
+	// Received bytes from worker within the time limit.
```

Comment:
nit: this line belongs to the `Ok(Ok(...))` match arm to explain what was meant by nested result.